sdk/core/core-http/review/core-http.api.md

```diff
@@ -23,6 +23,14 @@ export interface AccessTokenCache {
     setCachedToken(accessToken: AccessToken | undefined): void;
 }
 
+// @public
+export class AccessTokenRefresher {
+    constructor(credential: TokenCredential, scopes: string | string[], requiredMillisecondsBeforeNewRefresh?: number);
+    // (undocumented)
```

Comment:
missing doc comment

Reply:
Thanks! Fixed!

I see more
![image](https://user-images.githubusercontent.com/10452642/90834064-395a2100-e2fe-11ea-9524-8e7cbbe63844.png)
I'll log an issue for this.

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/10734